src/Sylius/Bundle/ResourceBundle/Behat/WebContext.php

```diff
@@ -392,6 +392,8 @@ public function iAddAttribute($attribute)
         $addAttributesButton->press();
 
         $this->waitForModalToDisappear($attributesModalContainer);
+
+        $this->getSession()->wait(500);
```

Comment:
In fact, it can be useful for fix sometimes-failing attributes scenarios. @pjedrzejewski have pointed out this issue once.